.github/workflows/build-and-deploy-images.yml

```
@@ -40,24 +40,35 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and tag base-app image
+        working-directory: ./docker-static/base-app
+        run: |
+          docker build . -t lf-base-app
+          docker tag lf-base-app sillsdev/web-languageforge:base-app-latest
+
```

I think I'd like to build the base app image first, *then* log into Docker. That way if there are any errors logging in we'll still get to see the base app build happen, which will inform us of any build problems. The way it is right now, a temporary issue logging into Docker Hub could mask a build failure.

```suggestion
      - name: Build and tag base-app image
        working-directory: ./docker-static/base-app
        run: |
          docker build . -t lf-base-app
          docker tag lf-base-app sillsdev/web-languageforge:base-app-latest

      - name: Log in to Docker Hub
        uses: docker/login-action@v1
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_PASSWORD }}
```